docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -591,63 +594,119 @@ plugin libraries, composed of:
   - It also implements OCALLs for the enclave-side library to access
   host-side services for SGX quote verification.
 
-The OE SDK V0.10 release will also have enclave-side and host-side plugins
-for verification of SGX ECDSA reports generated by the legacy API function
-`oe_get_report()` and raw SGX ECDSA quotes generated by the Intel SGX SDK.
-These two evidence formats are identified by their UUIDs:
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`: for SGX ECDSA report generated
-by `oe_get_report()`.
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
-by the Intel SGX SDK.
+Currently there is no plan to implement verifier plugins for verification of
+EPID evidence. Verification of SGX EPID quotes is not supported by
+the Intel SGX SDK. In all existing SGX EPID solutions, EPID quotes are sent to
+Intel Attestation Service (IAS) backend for verification.
+
+The OE SDK V0.1x release will also have enclave-side and host-side plugins
+for verification of selected types of OE reports for SGX generated by the legacy
+API function `oe_get_report()` and selected types of SGX quotes generated by
+the Intel SGX SDK DCAP and quote-ex libraries. Note: the plan is for
+the attestation API function `oe_verify_evidence()` to only support a minimum
+set of legacy OE reports and SGX quotes as needed. There is no intention for
+`oe_verify_evidence()` to support verification of all possible types of legacy
+OE reports and SGX quotes.
+
+- `OE_FORMAT_UUID_LEGACY_REPORT_REMOTE`
+  - For OE reports generated by the legacy API function `oe_get_report()`
+  with the `OE_REPORT_FLAGS_REMOTE_ATTESTATION` flag.
+  - Note: no plan to provide verifier plugin for verification of OE reports
```

Comment:
I understand that the primary focus of the current work is to enable the attestation service scenarios, which are necessarily remote attestation. However, I would strongly suggest planning to implement verifier plugin for the legacy formats as part of the deprecation plan for the OE legacy APIs. 

i.e. it's probably cheaper to implement the verification plugin wrapper, deprecate the old `oe_get_report()` API and then removing the legacy verification plugin, than it is to continue to support the old reporting APIs.

Reply:
The only legacy OE report format is the output of `oe_get_report()` for SGX local attestation, and this will not one of the attestation service scenarios. We already have the legacy API `oe_verify_report()` for it.

I think that, when the plugin APIs are out of experimental, it's better to deprecate `oe_get_report()` and `oe_verify_report()` soon, instead of adding a plugin for OE report for SGX local attestaion.

Reply:
I tend to agree with Simon on this one.

Reply:
I think we can explore various options when we are ready to deprecate `oe_get_report()`. As Simon mentioned, I'd like this PR to focus on enabling the attestation service scenarios.